### DISCUSSION TITLE
[chrome] chrome.action is missing the method openPopup and the type OpenPopupOptions

### DISCUSSION THREAD

**Discussion by jkeegan2** [DefinitelyTyped/DefinitelyTyped#60529] [Issues with a @types package]
chrome.action contains a property openPopup(options?: OpenPopupOptions, callback?: () => void),

where OpenPopupOptions has the following shape:
{
    windowId?: number,
}

as per the documentation here: https://developer.chrome.com/docs/extensions/reference/action/#method-openPopup
and here: https://developer.chrome.com/docs/extensions/reference/action/#type-OpenPopupOptions

[a comment by an automated account is not shown]

**Comment by Grayson-code** [comment]
fixed in #60870
